how-to/register-with-dock/client/src/dock.ts

```diff
@@ -89,6 +89,30 @@ export function dockGetCustomActions(): CustomActionsMap {
 			if (payload.callerType === CustomActionCallerType.CustomButton && currentView) {
 				await removeFromFavorites();
 			}
+		},
+		"custom-menu": async (payload): Promise<void> => {
+			// The custom-menu is triggered from the dock
+			if (payload.callerType === CustomActionCallerType.CustomButton) {
+				const res = await showCustomMenu({ x: 330, y: 34 }, payload.windowIdentity, [
```

Comment:
Is there any way of knowing where the call came from x and y wise rather than hardcoded?

Reply:
i.e. this is from the bounds of the dock window using parent identity but it is assuming the position of the button. If the length of buttons changes or the user repositions the buttons using drag and drop then this will not show up in the correct place. Maybe a feature request if the action position isn't available?

Reply:
You do get the x,y where the click happened, and I had some funky calculations to work it out, I could reinstate that https://github.com/built-on-openfin/workspace-starter/pull/520/commits/2da59c1c970cdf828938f0592296bfd29b5a79fb#diff-f81b45c38a456a3cc79d4c932828bcd4ec4eeb2abb628f7aa7886c41995766abL409